### PR TITLE
Replace `logAndThrow` with `throw new Error`

### PR DESCRIPTION
**Pull request by mykola-mokhnach**
logAndThrow leaves unnecessary traces in logs, which confuse users when these are ignored in the caller code.

### CODE REVIEW

File: lib/helpers.js [hunk 1]
```diff
@@ -249,7 +249,7 @@ function buildStartCmd (startAppOptions, apiLevel) {
 const getSdkToolsVersion = _.memoize(async function getSdkToolsVersion () {
   const androidHome = process.env.ANDROID_HOME;
   if (!androidHome) {
-    log.errorAndThrow('ANDROID_HOME environment varibale is expected to be set');
+    throw new Error('ANDROID_HOME environment varibale is expected to be set');
```
Comment (imurchie) [inline lib/helpers.js]: variable

File: lib/tools/android-manifest.js [hunk 2]
```diff
@@ -307,13 +301,13 @@ manifestMethods.insertManifest = async function (manifest, srcApk, dstApk) {
  * @return {boolean} True if the manifest requires Internet access permission.
  */
 manifestMethods.hasInternetPermissionFromManifest = async function (localApk) {
+  await this.initAapt();
+  log.debug("Checking if has internet permission from manifest");
```
Comment (imurchie) [inline lib/tools/android-manifest.js]: Grammar.

File: lib/helpers.js (hunk 1 above)
Comment (mykola-mokhnach) [reply lib/helpers.js]: fixed

File: lib/tools/android-manifest.js (hunk 2 above)
Comment (mykola-mokhnach) [reply lib/tools/android-manifest.js]: fixed

**Pull request merged**